src/main/java/me/senseiwells/essentialclient/utils/misc/RecipeHelper.java

```diff
@@ -12,4 +13,13 @@ public static ItemStack getOutput(Recipe<?> recipe) {
 		//$$return recipe.getOutput();
 		//#endif
 	}
+
+	public static ItemStack getOutput(StonecuttingRecipe recipe) {
```

Comment:
Why have you overloaded this?
Can you not just pass it into `getOutput(Recipe<?> recipe)`?

Reply:
Yes, you can't do it since StonecuttingRecipe is not directly extended class of Recipe<?>

Reply:
Yes it is...


![image](https://user-images.githubusercontent.com/66843746/233806289-c7a29285-cfc8-446a-9731-9dcd2ef98bd4.png)
![image](https://user-images.githubusercontent.com/66843746/233806295-a23f3575-783f-40d5-a3bc-c7396506a343.png)

Reply:
It threw error before but it does not.... 🤷